pallets/transaction-multi-payment/src/lib.rs

```diff
@@ -121,6 +121,10 @@ pub mod pallet {
 		/// Accepted currency removed
 		/// [currency]
 		CurrencyRemoved(AssetId),
+
+		/// Transaction fee paid in non-native currency
+		/// [Account, Currency, Native fee amount, Non-native fee amount]
+		FeeTransferred(T::AccountId, AssetId, Balance, Balance),
```

Comment:
`FeeWithdrawn` is a better name as transfer event should always state destination as well

Reply:
done